WalletWasabi.Tests/UnitTests/Crypto/ZeroKnowledge/LinearRelationTests.cs

```diff
@@ -34,7 +35,7 @@ public void VerifyResponses(uint scalarSeed1, uint scalarSeed2)
 			var secretNonces = new ScalarVector(new Scalar(23), new Scalar(42));
 			var publicNonce = Enumerable.Zip(secretNonces, generators, (s, g) => s * g).Sum();
 			var challenge = new Scalar(101);
-			var response = equation.Respond(witness, secretNonces, challenge);
```

Comment:
these are the unit tests for the Equation class, i guess they are in the wrong file (it used to be named Equation.cs but i renamed it after it started covering all 3 LinearRelation classes)